src/ansys/fluent/core/solver/flobject.py

```diff
@@ -33,10 +33,10 @@
 
 
 class InactiveObjectError(RuntimeError):
-    """Provides the error when the object is inactive."""
+    """Raised on attempt to access an inactive object."""
```

Comment:
Class descriptions begin with verbs ending with "s" or "es".

Reply:
Then something like gets raised? Doesn't sound right.
cc. @seanpearsonuk

Reply:
What about: "Provides an error message on attempt to access an inactive object" ?

Reply:
Provides the error when an inactive object is accessed.

Reply:
@prmukherj either what you have written or
"Inactive object access"
```
>>> help(ValueError)
Help on class ValueError in module builtins:

class ValueError(Exception)
 |  Inappropriate argument value (of correct type).
...

>>> help(RuntimeError)
Help on class RuntimeError in module builtins:

class RuntimeError(Exception)
 |  Unspecified run-time error.
...

class TypeError(Exception)
 |  Inappropriate argument type.
```
etc

I don't think we actually want to say "Provides the error when..." as we have in the current code.